pkg/config/config_template.yaml

```diff
@@ -1579,6 +1579,18 @@ api_key:
 # extra_config_providers:
 #   - clusterchecks
 
+## @param autoconfig_exclude_features - list of comma separated strings - optional
+## Exclude features automatically detected and enabled by environment autodiscovery.
+## Supported syntax is a list of `(<attribute>:)<regexp>`. Currently only the `name` attribute is supported.
```

Comment:
The main idea is to support `.*` to exclude all features. I also expect the number of features to grow.
I'll add the list of features.

Reply:
> The main idea is to support `.*` to exclude all features.

Then a sentinel value (maybe `*` or `all`) would suffice, without any complex matching. Regular expressions have their uses, but for a small set of strings (even if it triples in size) that have little overlap they sure seem to me like an overkill.

And it would be great to have input validation for exclusion rules too to catch typos.

Reply:
It's indeed a bit overkill but it re-uses the same syntax as `ac_exclude/container_exclude/container_exclude_logs`, which is a good thing I believe.
However it's true that we don't need the regexp syntax for the `include` part as it would not make sense.